corehq/apps/reports/templates/reports/async/bootstrap5/filters.html

```diff
@@ -1,10 +1,10 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 <fieldset>
-  <legend><span data-bind="text: filterHeadingName">{% trans "Report Filters" %}</span></legend>
+  <legend class="fs-4"><span data-bind="text: filterHeadingName">{% trans "Report Filters" %}</span></legend>
```

Comment:
Won't "Report Filters" always be overwritten by `filterHeadingName`? Can you remove it?

(That's pre-existing, so feel free to ignore.)

Reply:
going to leave it